docker/calamari.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04 as builder
+LABEL description="run calamari binary distribution in docker"
+ARG TAG_NAME
+ARG BINARY="https://github.com/Manta-Network/Manta/releases/download/${TAG_NAME}/calamari-pc"
+ARG CALAMARI_GENESIS="https://raw.githubusercontent.com/Manta-Network/Manta/manta-pc/genesis/calamari-genesis.json"
+ARG KUSAMA_GENESIS="https://raw.githubusercontent.com/paritytech/polkadot/master/node/service/res/kusama.json"
```

Comment:
as above. we should reference a specific git sha or tag which identifies the specific version of the kusama chainspec to be used in the docker image, since `master` is again a tip/head pointer.